src/components/NcActions/NcActions.vue

```diff
@@ -1335,6 +1335,14 @@ export default {
 
 		opened() {
 			this.intersectIntoCurrentFocusTrapStack()
+
+			// Ensure that pressing escape will close the menu even if the menu is not hovered
+			// and not currently active, e.g. because user opened the context menu
+			if (this.opened) {
+				document.body.addEventListener('keydown', this.handleEscapePressed)
```

Comment:
The menu now closes on every keydown, not only on escape. I don't think that is desired, no?

Reply:
:see_no_evil: fixed